docs/samples/git-setup/policytemplates/version_4.Y.Z/sno-ran-du/sno-ran-du-pg-v4-Y-Z-v4-full-DU.yaml

```diff
@@ -0,0 +1,376 @@
+          pipelines: '{{hub fromConfigMap "" (printf "%s-pg" .ManagedClusterName) "cluster-log-fwd-pipelines"  | toLiteral hub}}'
+          filters: '{{hub fromConfigMap "" (printf "%s-pg" .ManagedClusterName) "cluster-log-fwd-filters"  | toLiteral hub}}'
+          serviceAccount:
+            name: collector
+    # TUNED CONFIG
+    - path: source-crs/TunedPerformancePatch.yaml    
+      patches:
+      - spec:
+          recommend:
+            - machineConfigLabels:
+                machineconfiguration.openshift.io/role: master
+              priority: 19
+              profile: performance-patchh
+          profile:
+            - name: performance-patch
+              data: |
+                [main]
+                summary=Configuration changes profile inherited from performance created tuned
+                include=openshift-node-performance-openshift-node-performance-profile
+                [bootloader]
+                cmdline_crash=-tsc=nowatchdog
+                cmdline_crash1=tsc=reliable
+                [sysctl]
+                kernel.timer_migration=1
+                kernel.sysrq=1
+                kernel.panic_on_rcu_stall=1
+                kernel.hung_task_panic=1
+                [scheduler]
+                group.ice-ptp=0:f:10:*:ice-ptp.*
+                group.ice-gnss=0:f:10:*:ice-gnss.*
+                [service]
+                service.stalld=start,enable
+                service.chronyd=stop,disable
+    # MACHINE CONFIG
+    - path: source-crs/MachineConfigGeneric.yaml
+      complianceType: mustonlyhave # This is to update array entry as opposed to appending a new entry.
+      patches:
+      - metadata:
+          name: 02-master-workload-partitioning
+        spec:
+          config:
+            storage:
+              files:
+                - contents:
+                    # crio cpuset config goes below. This value needs to be updated and matched with PerformanceProfile. Check the link for more info on the content.
+                    source: '{{hub fromConfigMap "" (printf "%s-pg" .ManagedClusterName) "machine-config-storage-source-1" hub}}'
+                  mode: 420
+                  overwrite: true
+                  path: /etc/crio/crio.conf.d/01-workload-partitioning
+                  user:
+                    name: root
+                - contents:
+                    # openshift cpuset config goes below. This value needs to be updated and matched with crio cpuset (array entry above this). Check the link for more info on the content.
+                    source: '{{hub fromConfigMap "" (printf "%s-pg" .ManagedClusterName) "machine-config-storage-source-2" hub}}'
+                  mode: 420
+                  overwrite: true
+                  path: /etc/kubernetes/openshift-workload-pinning
+                  user:
+                    name: root
+- name: v4-sriov-config-policy
+  manifests:
+    # SRIOV
```

Comment:
For all of these we should not repeat any of the content already in the source CR.